README.md

```diff
@@ -77,3 +77,45 @@ Create your message in $message and
 ```
 $mailer->sendMessage($message);
 ```
+
+## Retrieving receivers from objects
+
+It is possible to use a Determiner to retrieve the receivers (email/name) from an email configuration.
+For instance from an object:
+
+``` php
+$profile = new Profile();
+$profile->setEmail('foobarz@example.com');
+$profile->setFullname('Foo Barz');
+
+$emailConfiguration = ['email' => '{{email}}', 'name' => '{{full_name'}}];
+$determiner = new RonRademaker\Mailer\Receiver\Determiner();
+$receiver = $determiner->getReceivers($profile, null, $emailConfiguration);
+// results in ['email' => 'foobarz@example.com', 'name' => 'Foo Barz'];
```

Comment:
If the input is of that type of format (like the Mailer expects) it will also output it in that format. I will update the documentation to represent that (and add a unit test for that scenario).